datasets/pcfg/tensorflow-datasets/pcfg_full_data.py

```diff
@@ -1,9 +1,14 @@
 """pcfg_full_data dataset."""
 
-from example_generator import example_generator
+import sys
+
 import tensorflow as tf
 import tensorflow_datasets as tfds
 
+sys.path.append('../../datasets')
```

Comment:
I know this is common in Python, but is there a cleaner way? (I don't know myself, so, just asking. If this is the only way, then it's fine)